pokete_classes/pokestats.py

```diff
@@ -0,0 +1,137 @@
+"""Contains classes nedded for pokete stats"""
+
+from datetime import datetime
+
+import scrap_engine as se
+
+from pokete_classes.hotkeys import Action
+from pokete_classes.loops import easy_exit_loop
+from pokete_classes.ui_elements import LabelBox
+
+
+class PokeStats:
+    """Holds a poketes statistics data
+    ARGS:
+        poke_name: The poketes name
+        ownership_date: The date of ownership
+        caught_by: The balls name the pokete got caught with
```

Comment:
I think `caught_by` is a bit misleading. When I first read it, I thought of the player who caught this pokete (because of trading; Good idea, actually. I would be happy to see this, but it is non-blocking).

Therefore, I would rather change this to `caught_with`.

Reply:
Thanks, renamed